commands/errors.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"strings"
+)
+
+const (
+	// NoTLSWarn Warning thrown when no SSL/TLS is used
+	NoTLSWarn = "WARNING! Communication is not secure, please consider using HTTPS. Letsencrypt.org offers free SSL/TLS certificates."
+)
+
+// checkTLSInsecure checks if the given gateway has https. Use tsInsecure to skip validations
+// Note: This function does not stop the flow only prints a warning
+func checkTLSInsecure(gateway string, tlsInsecure bool) string {
+	if !tlsInsecure {
+		if !strings.HasPrefix(gateway, "https") {
+			return (NoTLSWarn)
```

Comment:
We can remove the parentheses 
```suggestion
			return NoTLSWarn
```